sensor/common/compliance/command_handler_impl.go

```diff
@@ -45,7 +49,14 @@ func (c *commandHandlerImpl) Stop(_ error) {
 	c.stopper.Client().Stop()
 }
 
-func (c *commandHandlerImpl) Notify(common.SensorComponentEvent) {}
+func (c *commandHandlerImpl) Notify(e common.SensorComponentEvent) {
+	switch e {
+	case common.SensorComponentEventCentralReachable:
+		c.centralReachable.CompareAndSwap(false, true)
```

Comment:
Why not `Store`?

Reply:
It doesn't really make a difference, I changed it to Store so it's easier to read.